packages/postgres-database/src/simcore_postgres_database/utils_folders.py

```diff
@@ -428,6 +485,7 @@ async def _check_folder_and_access(
         FolderNotFoundError
         FolderNotSharedWithGidError
         InsufficientPermissionsError
+        UnexpectedFolderAccessError
     """
     folder_entry: int | None = await connection.scalar(
         sa.select([folders.c.id])
```

Comment:
below you have a huge comment about API DB layer. here you access the DB... not sure that comment makes any sense. It also shows how these kind of comments are misleading.